sale_promotion_rule_delivery/i18n/ca.po

```diff
@@ -0,0 +1,22 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* sale_promotion_rule_delivery
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 10.0\n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: 2021-03-26 16:46+0000\n"
+"Last-Translator: Daniel Martinez Vila <daniel.martinez@qubiq.es>\n"
+"Language-Team: none\n"
+"Language: ca\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: Weblate 4.3.2\n"
+
+#. module: sale_promotion_rule_delivery
+#: model:ir.model,name:sale_promotion_rule_delivery.model_sale_promotion_rule
+msgid "Sale Promotion Rule"
+msgstr "Regla de promoció de la venda"
```

Comment:
It's spanish. This file should be updated or removed.

Reply:
@acsonefho No, it's catalan.